packages/shopify-codemod/transforms/remove-empty-catch-blocks.js

```diff
@@ -0,0 +1,16 @@
+export default function removeEmptyCatchBlocks({source}, {jscodeshift: j}, {printOptions = {}}) {
+
+  return j(source)
+    .find(j.TryStatement, {
+      type: 'TryStatement',
+      handler: {
+        type: 'CatchClause',
```

Comment:
In that case I should probably update the other transforms to use `j.Type.name` instead of literals.